src/langchain_google_alloydb_pg/__init__.py

```diff
@@ -25,5 +26,6 @@
     "AlloyDBLoader",
     "AlloyDBDocumentSaver",
     "AlloyDBChatMessageHistory",
+    "pgvector_migrator",
```

Comment:
Let's remove this. We rather keep this in a module rather than exposing it at the top level, since it's not the main functionality.